packages/sd-jwt/__tests__/verify.test.ts

```diff
@@ -103,11 +133,22 @@ describe("Verify", () => {
     expect(op).rejects.toBeInstanceOf(Error);
   });
 
+  it("should fail if at least one disclosure fails to parse", () => {
+    const op = verify(sdjwt_with_wrong_disclosures_signed_with_key_a, {
+      jwksUri: wellKnownUrl("a-and-b"),
+    });
+    expect(op).rejects.toBeInstanceOf(Error);
+  });
+
+  it("should fail if at least one disclosure is missing in the SD-JWT", () => {
+    const op = verify(sdjwt_with_disclosures_signed_with_key_a_missing_claims, {
+      jwksUri: wellKnownUrl("a-and-b"),
+    });
+
```

Comment:
There seems to be a syntax error. Perhaps the parentheses should be closed here!